src/syntax_mapping.rs

```diff
@@ -172,6 +173,10 @@ impl<'a> SyntaxMapping<'a> {
         }
         None
     }
+
+    pub fn insert_ignored_suffix(&mut self, suffix: &'a str) {
```

Comment:
Since this is public API it makes sense to give some extra thought. I did that, and concluded this API matches the existing API (`fn insert`) as much as it can. Symmetry and predictability is nice in an API. Well done :)